meta/meta.go

```diff
@@ -292,32 +291,24 @@ func (m *Meta) DropDatabase(dbID int64) error {
 }
 
 // DropTable drops table in database.
-// If delAutoID is true, it will delete the auto_increment id key-value of the table.
-// For rename table, we do not need to rename auto_increment id key-value.
-func (m *Meta) DropTable(dbID int64, tblInfo *model.TableInfo, delAutoID bool) error {
+func (m *Meta) DropTable(dbID int64, tblID int64) error {
 	// Check if db exists.
 	dbKey := m.dbKey(dbID)
 	if err := m.checkDBExists(dbKey); err != nil {
 		return errors.Trace(err)
 	}
 
 	// Check if table exists.
-	tableKey := m.tableKey(tblInfo.ID)
+	tableKey := m.tableKey(tblID)
 	if err := m.checkTableExists(dbKey, tableKey); err != nil {
 		return errors.Trace(err)
 	}
 
 	if err := m.txn.HDel(dbKey, tableKey); err != nil {
 		return errors.Trace(err)
 	}
-
-	if delAutoID {
```

Comment:
Should we keep the auto-id key for renaming table operation？

Reply:
No.
We will use the new auto-id key after renaming table. So we can delete it.